components/CallForCounselors/Process.js

```diff
@@ -73,45 +73,48 @@ const Process = ({ featureKeyword, milestones }) => {
     <Main>
       <ProcessAndDatesGrid columns="repeat(auto-fit,minmax(320px,1fr))">
         <Cell>
-          <h3>Format and Application Process</h3>
-          <p>You can apply for the following audience and format.</p>
+          <h3>Talk Audiences and Formats</h3>
+          <p>You can apply to speak for the following audiences and formats:</p>
           <ProcessBlock>
             <ProcessAndDatesSubHeading>
-              Professional & Family
+              Professional & Family Formats
             </ProcessAndDatesSubHeading>
             <p>
               THAT Conference is a professional polyglot technical conference
-              that also has a rich experience for child geeklings. Your session
-              could be aimed at the professional track or for family geeklings.
+              that also has a rich experience for children and other family
+              geeklings. Your session could be aimed at the professional track
+              or for family geeklings.
```

Comment:
"family geeklings" is being used at the end of two sentences in a row - could the wording here be changed in one of them?

Reply:
Resolved